LoopKitUI/Extensions/UIColor+LoopColors.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  UIColor+LoopColors.swift
+//  LoopKitUI
+//
+//  Created by Nathaniel Hamming on 2020-01-27.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+extension UIColor {
+    public static var loopSelectable: UIColor {
+        return .systemBlue
+    }
+    
+    public static var loopSelected: UIColor {
+        return .systemBlue
+    }
+    
+    public static var loopNumbering: UIColor {
+        return .systemBlue
+    }
+    
+    public static var loopDim: UIColor {
+        if #available(iOSApplicationExtension 13.0, *) {
+            return .secondaryLabel
+        } else {
+            return .systemGray
+        }
+    }
+    
+    public static var loopDestructive: UIColor {
+        return .systemRed
+    }
+    
+    public static var loopLabel: UIColor {
+        if #available(iOSApplicationExtension 13.0, *) {
+            return .label
+        } else {
+            return .black
+        }
+    }
+    
+    public static var loopWhite: UIColor {
+        // some views, like the barcode scanners are styled with dark backgrounds even in light mode
```

Comment:
[nit] Not sure what this comment means because `loopWhite` == `.white`.

Reply:
Here I'm trying to explain that usually you would expect support for a dark and light colour, but the current designs require white color. I'll update to state this in code instead of a comment.

Reply:
updated to `loopAlwaysWhite`. I hope this gets across the point. Removed the comment.